components/Chips/src/MDCChipView.h

```diff
@@ -135,6 +135,16 @@
 @property(nonatomic, readwrite, setter=mdc_setAdjustsFontForContentSizeCategory:)
     BOOL mdc_adjustsFontForContentSizeCategory UI_APPEARANCE_SELECTOR;
 
+/**
+ Enable legacy font scaling curves for Dynamic Type.
+
+ Legacy font scaling uses the older [UIFont mdc_fontSizedForMaterialTextStyle:scaledForDynamicType:
+ category instead of the current MDCFontScaler API.
+
+ Default value is NO.
```

Comment:
Should the default value be YES? I remember in a meeting the other day the team seemed to decide that we should not introduce changes to clients at the same time we introduce changes to component. In other words, we should make changes we add to our library opt in, rather than opt out. Am I misremembering, or did I misunderstand that conversation?

Reply:
I agree. This flag here was to match the one in MDCButton. Do you prefer default it to YES?

Reply:
I don't really have a preference, I'm just trying to review the PR to the team's standards. I guess if it was okay to do it this way with buttons it's okay to do it here 🤷‍♂

Reply:
Is this a breaking change for clients? If so, please note that in your PR. I also agree that our team tends to opt-in to such changes. When this was done for Buttons, was it a breaking change for clients?

Reply:
I am going to change it to YES.

Reply:
Addressed in 71ab0082bf3ce8110fb628ecad329335108182fc. Thanks for the comments and discussion here!